content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -132,13 +159,28 @@ While Kubernetes does not provide a native solution for cluster-level logging, t
 
 ![Using a node level logging agent](/images/docs/user-guide/logging/logging-with-node-agent.png)
 
-You can implement cluster-level logging by including a _node-level logging agent_ on each node. The logging agent is a dedicated tool that exposes logs or pushes logs to a backend. Commonly, the logging agent is a container that has access to a directory with log files from all of the application containers on that node.
-
-Because the logging agent must run on every node, it's common to implement it as either a DaemonSet replica, a manifest pod, or a dedicated native process on the node. However the latter two approaches are deprecated and highly discouraged.
-
-Using a node-level logging agent is the most common and encouraged approach for a Kubernetes cluster, because it creates only one agent per node, and it doesn't require any changes to the applications running on the node. However, node-level logging _only works for applications' standard output and standard error_.
-
-Kubernetes doesn't specify a logging agent, but two optional logging agents are packaged with the Kubernetes release: [Stackdriver Logging](/docs/user-guide/logging/stackdriver) for use with Google Cloud Platform, and [Elasticsearch](/docs/user-guide/logging/elasticsearch). You can find more information and instructions in the dedicated documents. Both use [fluentd](http://www.fluentd.org/) with custom configuration as an agent on the node.
```

Comment:
Overall this is a useful set of changes. However, the content guidelines themselves are under discussion.

I wonder: would it be OK to leave in mentions of Elasticsearch, Stackdriver Logging, and fluentd, and then see how this PR looks?

Maybe reword the original text, maybe move the information to a What's Next? section. Also retain the relevant Task pages.

Reply:
@sftim 
> I wonder: would it be OK to leave in mentions of Elasticsearch, Stackdriver Logging, and fluentd, and then see how this PR looks?

Given this update from @aimeeu:
> Kubernetes doesn't specify a logging agent, and no logging agents are
packaged with the Kubernetes core release.

All concerns about third-party content aside: if logging agents aren't packaged with the release anymore, then I think the proposed changes make sense. I also think it makes sense to get feedback from @piosz 👋 and/or @kubernetes/sig-instrumentation-bugs.

Reply:
Did we ever actually ship an agent with releases? I think everything we had were external to the kube release itself. I think going with a neutral statement here is the right way to go.

Reply:
cc @coffeepac

Reply:
@brancz  When I wrote this PR, I wasn't sure about logging agents being packaged with Kubernetes, so I asked in the  #sig-docs channel. Timothy St. Clair [replied](https://kubernetes.slack.com/archives/C1J0BPD2M/p1567521075006200), "Logging is external from core and there are EFK helm charts that are community supported.  That falls into the external add-ons category and will never be part of k8s release."   So I decided to point users to the CNCF landscape for logging agents rather than mentioning specific ones.

Reply:
Yep.  I'm an owner of the ES addon and removing the reference to this stuff sounds correct to me as well.